Portfolio/index.html

```diff
@@ -6,8 +6,66 @@
     <link rel="stylesheet" href="index.css" />
   </head>
 
-  <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+  <body style="margin: 2; padding: 0">
+    <h1>Doe.</h1>
+    <section>
+      <ul>
+        <li>ABOUT</li>
+        <li>SKILLS</li>
+        <li>PROJECTS</li>
+        <li>REVIEW</li>
+      </ul>
+    </section>
+    <img src="/phone.icon.png" alt="phoneicon"><br><br>
+    <span>
+      <h4>JohanDoe</h4>
+      <img src="/johnDoe.png" alt="pic"><br>
+      <strong>Front End Developer</strong>
+    </span>
```

Comment:
The closing tag for the pre element is missing a forward slash. It should be </pre>.